src/module/migrator/versions/Version0_7_2.ts

```diff
@@ -0,0 +1,70 @@
+// TODO: How to trigger test migration.
+// TODO: How to test migration results?
+
+
+import {VersionMigration} from "../VersionMigration";
+import {SR5} from "../../config";
+import SR5ActorBase = Shadowrun.SR5ActorBase;
+
+/** NPC / Grunt feature set
+ * - Add npc character data.
+ * - Add track disabled feature
+ */
+export class Version0_7_2 extends VersionMigration {
+    get SourceVersion(): string {
+        return '0.7.1';
+    }
+
+    get TargetVersion(): string {
+        return Version0_7_2.TargetVersion;
+    }
+
+    static get TargetVersion(): string {
+        return '0.7.2';
+    }
+
+    static NoNPCDataForCharacter(actorData: SR5ActorBase): boolean {
+        return actorData.type === 'character' && (
+            actorData?.data?.is_npc === undefined ||
+            actorData?.data?.npc === undefined
+        );
+    }
+
+    static UnsupportedMetatype(actorData: SR5ActorBase): boolean {
+        const type = actorData.data.metatype.toLowerCase();
```

Comment:
still going to merge but leaving this comment to add later -- we should check and make sure `actorData.data.metatype` exists trying to access it. The problem is if it crashes on an undefined in here the whole migration might fail.